src/main/java/com/puppycrawl/tools/checkstyle/checks/javadoc/AbstractJavadocCheck.java

```diff
@@ -278,9 +311,13 @@ public final void visitToken(DetailAST blockCommentNode) {
             }
             else {
                 final ParseErrorMessage parseErrorMessage = result.getParseErrorMessage();
-                log(parseErrorMessage.getLineNumber(),
-                        parseErrorMessage.getMessageKey(),
-                        parseErrorMessage.getMessageArguments());
+                final boolean looseHtmlInJavadoc = "javadoc.unclosedHtml"
```

Comment:
@Vladlis @rnveach I can resolve this indirectly in the following way:

```
                 processTree(result.getTree());
             }
             else {
-                final ParseErrorMessage parseErrorMessage = result.getParseErrorMessage();
-                final boolean looseHtmlInJavadoc = "javadoc.unclosedHtml"
-                        .equals(parseErrorMessage.getMessageKey());
+                final DetailNode javadocTree = result.getTree();
 
-                if (looseHtmlInJavadoc && doesCheckAcceptJavadocWithNonTightHtml()) {
-                    processTree(result.getTree());
+                if (javadocTree != null && doesCheckAcceptJavadocWithNonTightHtml()) {
+                    processTree(javadocTree);
                 }
 
-                if (!looseHtmlInJavadoc || violateExecutionOnNonTightHtml) {
+                if (javadocTree == null || violateExecutionOnNonTightHtml) {
+                    final ParseErrorMessage parseErrorMessage = result.getParseErrorMessage();
                     log(parseErrorMessage.getLineNumber(),
                             parseErrorMessage.getMessageKey(),
```

Please confirm if this would be acceptable. Thank you.